llvm/test/Transforms/LoopVectorize/RISCV/vplan-vp-intrinsics.ll

```diff
@@ -0,0 +1,133 @@
+; REQUIRES: asserts
+
+; RUN: opt -passes=loop-vectorize -debug-only=loop-vectorize \
+; RUN: -force-tail-folding-style=data-with-evl \
+; RUN: -prefer-predicate-over-epilogue=predicate-dont-vectorize \
+; RUN: -mtriple=riscv64 -mattr=+v -riscv-v-vector-bits-max=128 -disable-output < %s 2>&1 | FileCheck --check-prefixes=IF-EVL,CHECK %s
+
+; RUN: opt -passes=loop-vectorize -debug-only=loop-vectorize \
+; RUN: -force-tail-folding-style=none \
+; RUN: -prefer-predicate-over-epilogue=predicate-dont-vectorize \
+; RUN: -mtriple=riscv64 -mattr=+v -riscv-v-vector-bits-max=128 -disable-output < %s 2>&1 | FileCheck --check-prefixes=NO-VP,CHECK %s
+
+define void @foo(ptr noalias %a, ptr noalias %b, ptr noalias %c, i64 %N) {
+; IF-EVL: VPlan 'Initial VPlan for VF={vscale x 1,vscale x 2,vscale x 4},UF>=1' {
```

Comment:
> UF>=1 or UF=1? (https://reviews.llvm.org/D99750?id=558054#inline-1551940)

We are quite close to explicit unrolling, so UF should be made explicit soon. Would be good to still indicate the valid range of UFs.

Reply:
Am, not the problem of this patch. This is how the Vplan is printed, separate problem.

Reply:
When introducing the EVL, the transform should do something like `Plan->setUF(1)`